src/services/accounts/AccountsValidateService.ts

```diff
@@ -15,21 +15,29 @@
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import { hexToU8a, isHex } from '@polkadot/util';
-import { base58Decode, checkAddressChecksum } from '@polkadot/util-crypto';
+import {
+	allNetworks,
+	base58Decode,
+	checkAddressChecksum,
+} from '@polkadot/util-crypto';
 import { defaults } from '@polkadot/util-crypto/address/defaults';
 
 import { IValidateAddrResponse } from '../../types/responses/ValidateAddress';
 import { AbstractService } from '../AbstractService';
 
 export class AccountsValidateService extends AbstractService {
 	/**
-	 * Takes a given address and determines whether it is a ss58 formatted address,
-	 * and what the ss58 prefix for that address is.
+	 * Takes a given address and determines whether it is a ss58 or
+	 * a hex (from a u8 array) formatted address,
+	 * what the ss58 prefix for that address is,
+	 * what is the network address format,
+	 * and what the account ID is for this address.
 	 *
 	 * @param address ss58 or hex address to validate
 	 */
 	validateAddress(address: string): IValidateAddrResponse {
 		let u8Address;
+		let network;
```

Comment:
No need to have `let network` at the top level by the looks of it; just have `let network = null` on line 54, so it's more obvious at a glance where the variable is used :)

Reply:
done! thank you!